tap_jira/streams.py

```diff
@@ -2986,6 +2986,39 @@ class IssueStream(JiraStream):
         Property("updated", StringType),
     ).to_dict()
 
+    def get_url_params(
```

Comment:
Hey @prakharcode! Is this the only that supports start and end date?

Reply:
I had a use case specific for this but I explored where we can potentially use a new config `project` that can be used else where. I can get back with other filter cases but still would like to open atomic PRs if that fine?